R/agglomerate.R

```diff
@@ -226,6 +226,7 @@ setMethod("agglomerateByRank", signature = c(x = "SummarizedExperiment"),
         # adjust rownames
         rownames(x) <- getTaxonomyLabels(x, empty.fields, ...,
                                         with_rank = FALSE, resolve_loops = FALSE)
+        rownames(x) <- sort(rownames(x))
```

Comment:
Basically what this does is that it just mixes up the rownames. The data inside is not ordered, only the names are changed.

To get the desired behavior, add to the end of this function something like this

`x <- x[ order(rownames(x), ]`

Reply:
ACK